css/stable.css

```diff
@@ -8,23 +12,59 @@
   -moz-user-focus: normal;
   -moz-user-input: enabled;
 }
-.stable td {font: menu; white-space: nowrap; padding: 1px}
-.stable td:first-child div {
-  display: flex;
-  align-items: center;
-  gap: 0.35rem;
+.stable td {
+  font: menu;
+  font-family: inherit;
+  font-size: inherit;
+  white-space: nowrap;
+  padding: 0.15rem;
```

Comment:
Consider `padding: 0.15rem 0.25rem;`

Reply:
Just for conformation, perhaps you mean `padding: 0.25rem 0.15rem` if we're going to increase the top and bottom padding a bit?

Reply:
Yes, that is correct.